zuul.d/adoption.yaml

```diff
@@ -125,6 +125,13 @@
       - OWNERS
       - .*/*.md
 
+- job:
+    name: cifmw-data-plane-adoption-osp-17-to-extracted-crc-minimal-no-ceph
+    parent: cifmw-data-plane-adoption-osp-17-to-extracted-crc
+    vars:
+      use_ceph: "false"
```

Comment:
Why using a string for a boolean?

Reply:
ultimately this var is passed to a bash script, so it ends up being consumed as a string anyway, but I can change it to a bool, it should not matter for the use case